style.css

```diff
@@ -356,59 +367,1108 @@ span {
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  margin-top: 0.5rem;
+  border: none;
+}
+
+.form-btn:hover {
+  background: #fff;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+.form-btn :checked {
+  background: #fff;
+  border: 2px solid #1a2236;
+}
+
+.get {
+  color: #98dede;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 0.7rem;
+  margin-right: 0.5rem;
+}
+
+.get > li:nth-child(1) {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.get > li:nth-child(2) {
+  margin-top: 1rem;
+  font-size: 2rem;
+}
+
+.contact-footer {
+  display: flex;
+  flex-direction: column;
+  background: #1a2236;
+  width: 100%;
+  height: 150px;
 }
 
-@media screen and (min-width: 768px) {
+.contact-social {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  gap: 1rem;
+  justify-content: space-around;
+  margin-right: 2rem;
+}
+
+.project-card-d1,
+.project-d-header,
+.img1,
+.img2,
+.headline-p-d {
+  display: none;
+}
+
+.contact-logo1-d,
+.contact-logo2-d {
+  display: none;
+}
+
+@media screen and (min-width: 500px) and (max-width: 767px) {
+  .headline-p-d {
+    display: none;
+  }
+
   .headline-p {
     width: 100%;
-    height: 10%;
+  }
+
+  .project-cards {
+    display: none;
+  }
+
+  .project-d-header {
+    display: none;
+  }
+
+  .projects-d-container {
+    display: none;
+  }
+
+  .headline-img {
+    background-size: 100% 100%;
   }
 
   .headline-btn,
+  .form-btn,
   .aboutme-btn {
     width: 95%;
   }
 
+  .aboutme-techs,
+  .aboutme-framew,
+  .aboutme-skills {
+    width: 100%;
+    margin-right: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .aboutme-cards {
+    padding-right: 2.5rem;
+  }
+
+  .aboutme-p,
+  .contact-para {
+    width: 100%;
+  }
+
+  .logo-corner1,
+  .logo-corner,
+  .logo-corner2 {
+    left: calc(77.5% + 2rem);
+  }
+
+  .contact-form {
+    width: 100%;
+  }
+
+  .contact-footer {
+    width: 100%;
+  }
+
+  .contact input[type="text"] {
+    width: 100%;
+  }
+
+  .techs,
+  .framew {
+    padding-right: 10%;
+  }
+
+  .project-card-d1,
+  .project-d-header,
+  .img1,
+  .img2,
+  .headline-p-d {
+    display: none;
+  }
+
+  .contact1-img,
+  .contact2-img {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 769px) {
+  .header-m > .logo,
+  .menu {
+    display: none;
+  }
+
+  .header-m {
+    justify-content: center;
+  }
+
+  .menu-d-list {
+    list-style: none;
+    display: flex;
+    gap: 1rem;
+  }
+
+  .menu-d-list > li {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+  }
+
+  .headline {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .headline-d {
+    display: flex;
+    flex-direction: column;
+    padding: 6% 0 15% 0;
```

Comment:
- [ ] Please 🙏🏽 correct the padding of the title, paragraph, image size, and button to be similar to the Figma design I recommend using px instead of percentages or rem.

| Figma design | Your design |
|---|---|
| ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/754b798c-f95a-48f6-bddc-0d0dabad2ff2) | ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/7ef0d613-89fd-4766-a59d-bd45562536db) |